Wikipedia/Categories/UIViewController+WMFOpenExternalUrl.m

```diff
@@ -0,0 +1,51 @@
+//  Created by Monte Hurd on 9/24/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "UIViewController+WMFOpenExternalUrl.h"
+#import "SVModalWebViewController.h"
+
+#import "Global.h"
+#import "ZeroConfigState.h"
+#import "SessionSingleton.h"
+#import "UIAlertView+BlocksKit.h"
+
+@implementation UIViewController (WMFOpenExternalLinkDelegate)
+
+- (void)wmf_openExternalUrl:(NSURL*)url {
+    NSParameterAssert(url);
+
+    void (^ openExternalURL)(NSURL*) = ^void (NSURL* url) {
```

Comment:
NBD, but breaking this blocks out into separate category methods might make the enclosing method easier to grok. IOW, the trade-off of capturing 1 argument (`url`) isn't worth the readability sacrifice of having all this inline blocks for things that could easily be instance methods w/ a URL argument.